Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -739,3 +944,40 @@ class MessageGenerator {
         }
     }
 }
+
+// The logic for this check comes from google/protobuf; the C++ and Java generators specificly.
+//
+// This is a helper for generating isInitialized methods.
+fileprivate func HasRequiredFields(_ fieldDesc: Google_Protobuf_FieldDescriptorProto, context: Context) -> Bool {
```

Comment:
This sucked.  As `swiftc` then got grumpy about the local with the same name, so I went ahead and renamed it changing the calling a little.